pkg/operator/resources/validations.go

```diff
@@ -102,7 +102,8 @@ func ValidateClusterAPIs(apis []userconfig.API, models *[]spec.CuratedModelResou
 	for i := range apis {
 		api := &apis[i]
 		if api.Kind == userconfig.RealtimeAPIKind || api.Kind == userconfig.BatchAPIKind {
-			if err := spec.ValidateAPI(api, models, projectFiles, types.AWSProviderType, config.AWS, config.K8s); err != nil {
+			models := []spec.CuratedModelResource{}
```

Comment:
Is it still necessary to have this here?

Reply:
`ValidateAPI` is also called for local deployments. When deploying locally, the `models` argument is required for caching the models locally in the Cortex cache higher up in the call stack.

But here, in `operator/resources/validations.go`, it isn't required. Still, were passing it in, because otherwise, each predictor validation would panic because `*models` can't be set. We could verify if `models != nil` though. What do you think?